modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,6 @@
+a-maximum-of-x-organizations-is-allowed=A maximum of {0} organizations is allowed.
+no-root-organization-found=No root organization found.
+select-organizations=Select organisations
```

Comment:
value and key are not consistent (organi**z**ation)

Reply:
Typo, gonna fix.